src/bin/openst.js

```diff
@@ -7,20 +7,59 @@ const program = require('commander');
 const connected = require('../connected');
 const OpenST = require('../openst');
 const { version } = require('../../package.json');
+const logger = require('./../logger');
 
 program
   .version(version)
-  .name('bt')
-  .arguments('openst <config> <organization> <eip20Token>')
+  .name('openst')
+  .description('An executable to setup OpenST.');
+
+program.command('openst <config> <eip20Token> <organization> ')
   .description('An executable to setup OpenST.')
   .action(
-    async (config, organization, eip20Token) => {
+    async (config, eip20Token, organization) => {
       await connected.run(
         config,
         async (chainConfig, connection) => {
-          const openst = new OpenST(chainConfig, connection);
-          await openst.setupOpenst(organization, eip20Token);
-          chainConfig.write(config);
+          try {
+            const openst = new OpenST(chainConfig, connection);
+            await openst.setupOpenst(eip20Token, organization);
```

Comment:
setupOpenst function expects organization first and then organization. Please align everywhere.
Since in the command line argument eip20Token is first and than organization, it might make sense to update function arguments accordingly.